plugins/client/grpc/client_config.go

```diff
@@ -73,6 +73,9 @@ func (c *Client) loadConfig() (*[]grpc.DialOption, error) {
 		return err
 	})
 
+	// using round-robin load balancer
+	options = append(options, grpc.WithDefaultServiceConfig(`{"loadBalancingPolicy":"round_robin"}`))
```

Comment:
Is this the routine rule for https://github.com/apache/skywalking-satellite/pull/68/files#r714050900?

Reply:
If so, it is OK for now, but we will need our own LB-policy implementation later. Because this is not real protocol-oriented `round_robin`.

Reply:
That's true. I think it is enough for us to specify a load-balance method.